code/unclassified/src/apriori/readme.md

```diff
@@ -0,0 +1,13 @@
+# cosmos
+> Your personal library of every algorithm and data structure code that you will ever encounter
+
+# Apriori Algoithm
```

Comment:
Please add more details to this algorithm as per our [documentation guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md).

Reply:
Added a brief description of the algorithm and time complexity.